src/EditorFeatures/Core/Implementation/CodeActions/CodeActionEditHandlerService.cs

```diff
@@ -233,6 +233,12 @@ internal class CodeActionEditHandlerService : ForegroundThreadAffinitizedObject,
                 return null;
             }
 
+            if (changedDocuments.Any(id => newSolution.GetDocument(id).HasInfoChanged(oldSolution.GetDocument(id))) ||
+                changedAdditionalDocuments.Any(id => newSolution.GetDocument(id).HasInfoChanged(oldSolution.GetDocument(id))))
+            {
+                return null;
+            }
```

Comment:
FYI: There is a SolutionChanged struct (or something like that) which is returned when you diff two solutions.  We probably need "RenamedDocuments" to be part of that.